test/e2e/kn.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 The knative Authors
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"testing"
+)
+
+type kn struct {
+	t         *testing.T
+	namespace string
+	l         Logger
+}
+
+func (k kn) run(args []string) string {
```

Comment:
we should make methods (run, runWithOpts) public as they represent public functionality of consumers of kn class. same for kubectl.

Reply:
OK